src/items.rs

```diff
@@ -1358,9 +1358,10 @@ fn rewrite_fn_base(context: &RewriteContext,
             FnArgLayoutStyle::BlockAlways => false,
             _ => {
                 // If we've already gone multi-line, or the return type would push
-                // over the max width, then put the return type on a new line.
+                // over the max width (including the whitespace after result),
```

Comment:
the white space i'm referring to is
if `ret_should_indent` is true 
`ln 1381: result.push('\n');`
or false
`ln 1385: result.push(' ');`  

which appended to result, so i think its after. lmk if i'm mistaken